### ISSUE TITLE
[BUG]Downloaders page not accessible for screen reading users

### ISSUE THREAD

**Issue by jessicarose** [common-voice/common-voice#5077]
**From community member email**
I found the website that we are supposed to download the dataset to be quite not accessible for screen readers.

https://commonvoice.mozilla.org/en/datasets

More precisely, I can't select what I actually want to download.
The list of languages is quite good, however, once I am in the page selecting any of the delta versions or the corpus is just not accessible.
I have tested on both JAWS and NVDA, which are the two most popular screen readers for windows.

**To Reproduce**
1. Go to https://commonvoice.mozilla.org/en/datasets with a screen reader enabled
2. Click on any dataset option for download

**Expected behavior**
User should be able to download corpus while navigating with screen reader

**Screenshots**

**Desktop or Mobile (please complete the following information):**

 **Additional Hardware (were you using headphones, an external speaker or an external microphone?):**
 - Type:
 - Model:

**Additional context**

**Comment by moz-bozden** [comment]
This feature has been removed from Common Voice and moved to MDC.
Closing...
